src/main/java/org/assertj/core/error/AssertionErrorMessagesAggregator.java

```diff
@@ -14,9 +14,9 @@
 
 import java.util.List;
 
-public class AssertionErrorMessagesAggregrator {
+public class AssertionErrorMessagesAggregator {
```

Comment:
Not being in the internal package, this class might be used by other projects/libraries and the renaming would be a breaking change. We should probably deprecate this one and create a new class with the correct namings.

@joel-costigliola do you agree?

Reply:
I agree!